pkg/storaged/utils.js

```diff
@@ -1050,3 +1090,31 @@ export function is_mounted_synch(block) {
 export function for_each_async(arr, func) {
     return arr.reduce((promise, elt) => promise.then(() => func(elt)), Promise.resolve());
 }
+
+/*
+ * Get mount points for a given org.freedesktop.UDisks2.Filesystem object
+ *
+ * This generalises getting the given MountPoints of a Filesystem for btrfs and
+ * other filesystems, for btrfs we want to know if a subvolume is mounted
+ * anywhere. UDisks is currently not aware of subvolumes and it's Filesystem
+ * object gives us the MountPoints for all subvolumes while we want it per
+ * subvolume.
+ *
+ * @param {Object} block_fsys
+ * @param {Object|null} subvol
+ * @returns {Array} an array of MountPoints
+ */
+export function get_mount_points(client, block_fsys, subvol) {
+    let mounted_at = [];
+
+    if (subvol) {
+        const btrfs_volume = client.blocks_fsys_btrfs[block_fsys.path];
```

Comment:
`block_fsys` might be null, no?

Reply:
Ack, the else does check for it so let's do `subvol && block_fsys`